Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -57,14 +63,18 @@ public struct OfflineMapAreasView: View {
                 .textCase(nil)
             }
             .task {
-                await makePreplannedMapModels()
+                await mapViewModel.requestUserNotificationAuthorization()
             }
             .task {
-                await mapViewModel.requestUserNotificationAuthorization()
+                await makePreplannedMapModels()
+                isConnected = networkMonitor.isConnected
```

Comment:
I feel like the if-else in the async `makePreplannedMapModels` method should be kept, while setting `isConnected = networkMonitor.isConnected` goes before the async call.